public/locales/en/translation.json

```diff
@@ -286,5 +286,194 @@
+      "Auburn": "Auburn",
+      "Black": "Black",
+      "Blonde": "Blonde",
+      "BlondeGolden": "Golden blonde",
+      "Brown": "Brown",
+      "BrownDark": "Dark brown",
+      "Platinum": "Platinum",
+      "Red": "Red"
+    },
+    "facialHairType": {
+      "label": "Facial hair type",
+      "Blank": "Blank",
+      "BeardMedium": "Medium beard",
+      "BeardLight": "Light beard",
+      "BeardMajestic": "Majestic beard",
+      "MoustacheFancy": "Fancy moustache",
+      "MoustacheMagnum": "Magnum moustache"
+    },
+    "skinColor": {
+      "label": "Skin color",
+      "Tanned": "Tanned",
+      "Yellow": "Yellow",
+      "Pale": "Pale",
+      "Light": "Light",
+      "Brown": "Brown",
+      "DarkBrown": "Dark brown",
+      "Black": "Black"
+    },
+    "eyeType": {
+      "label": "Eye type",
+      "Close": "Closed",
+      "Cry": "Crying",
+      "Default": "Default",
+      "Dizzy": "Dizzy",
+      "EyeRoll": "Rolling eyes",
+      "Happy": "Happy",
+      "Hearts": "Hearts",
+      "Side": "Look sideways",
+      "Squint": "Squint",
+      "Surprised": "Surprised",
+      "Wink": "Wink",
+      "WinkWacky": "Wacky wink"
+    },
+    "mouthType": {
+      "label": "Mouth type",
+      "Concerned": "Concerned",
+      "Default": "Default",
+      "Disbelief": "Disbelief",
+      "Eating": "Eating",
+      "Grimace": "Grimace",
+      "Sad": "Sad",
+      "ScreamOpen": "Screaming",
+      "Serious": "Serious",
+      "Smile": "Smile",
+      "Tongue": "Tongue out",
+      "Twinkle": "Twinkle",
+      "Vomit": "Vomit"
+    },
+    "topType": {
+      "label": "Top type",
+      "Eyepatch": "Eyepatch",
+      "Hat": "Hat",
+      "Hijab": "Hijab",
+      "LongHairBigHair": "Long, wavy hair",
+      "LongHairBob": "Bob",
+      "LongHairBun": "Bun",
+      "LongHairCurly": "Long, curly hair",
+      "LongHairCurvy": "Long, curvy hair",
+      "LongHairDreads": "Dreadlocks",
+      "LongHairFrida": "Frida Kahlo",
+      "LongHairFro": "Afro",
+      "LongHairFroBand": "Afro with headband",
+      "LongHairMiaWallace": "Mia Wallace",
+      "LongHairNotTooLong": "Medium length hair",
+      "LongHairShavedSides": "Long hair with sidecut",
+      "LongHairStraight": "Straight, long hair 1",
+      "LongHairStraight2": "Straight, long hair 2",
+      "LongHairStraightStrand": "Straight, long hair 3",
+      "NoHair": "No hair",
+      "ShortHairDreads01": "Short dreads 1",
+      "ShortHairDreads02": "Short dreads 2",
+      "ShortHairFrizzle": "Frizzle",
+      "ShortHairShaggyMullet": "Shaggy Mullet",
+      "ShortHairShortCurly": "Short, curly hair",
+      "ShortHairShortFlat": "Short, flat hair",
+      "ShortHairShortRound": "Short, round hair",
+      "ShortHairShortWaved": "Short, wavy hair",
+      "ShortHairSides": "Half-bald",
+      "ShortHairTheCaesar": "The Caesar 1",
+      "ShortHairTheCaesarSidePart": "The Caesar 2",
+      "Turban": "Turban",
+      "WinterHat1": "Winter hat 1",
```

Comment:
1 was? Äpfel? Birnen? 🤪

Reply:
Hab jetzt mal überall ne Raute vorne drangehängt.
"The Caesar #1", "The Caesar #2", etc.

Reply:
Danke, war eigentlich 1 Joke 😅